src/NetTopologySuite.Lab/Hull/ChiShape.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using NetTopologySuite.Geometries;
+using NetTopologySuite.Operation.Linemerge;
+using NetTopologySuite.Triangulate;
+using NetTopologySuite.Triangulate.QuadEdge;
+
+namespace NetTopologySuite.Hull
+{
+    public class ChiShape
+    {
+        private readonly Geometry _geom;
+        private readonly double _tolerance;
+
+        public ChiShape(Geometry geom, double tolerance)
+        {
+            _geom = geom;
+            _tolerance = tolerance;
```

Comment:
Probably also validate that `tolerance` is a finite, positive number.

Reply:
Check!